### DISCUSSION TITLE
Permanent scrolling in game

### DISCUSSION THREAD

**Discussion by Ossiost** [LizardByte/community#442] [Sunshine] [answered]
Hello everyone,

I have the following problem with a game (FM24 from fm-zocker).
If I scroll down or up only once in one of the lists in the game, it seems as if this command is sent permanently. It then always scrolls up or down so that I can no longer stop the scrolling.
If I am not scrolling in the game, I can stop the scrollbar at any point on the page.
I have no problem with this on Windows itself.
If I run the game directly on the computer and operate it with a mouse, it also works without any problems.

Does anyone have any ideas what else I can try?

Thank you very much

**Comment by grejfrutowypingwin** [comment]
I have the same problem. Reproducible in Kingdom Come 1 and Assassins Creed Black Flag so far. different mouses give the same behavior. Sunshine is hosted on windows, Moonlight also on Windows. Disabling high res scrolling does not help. I can provide debug logs. I'll test if chromecast gives the same behavior

**Reply by grejfrutowypingwin** [reply, marked as answer]
OK, visit discord, this topic was addressed overthere. Connect any mouse to host solves the issue.

[a comment by an automated account is not shown]
